frontend/src/actionCreators/parcelsActionCreator.ts

```diff
@@ -126,7 +126,7 @@ export const createParcel = (parcel: IParcel) => async (dispatch: Function) => {
       parcel,
     );
     dispatch(success(actionTypes.ADD_PARCEL, status));
-    dispatch(storeParcelDetail(null));
+    dispatch(storeParcelDetail(data));
```

Comment:
I know I've been back and forth on this a few times, but it really is convenient to jump to whatever parcel you just added/updated, especially with the new subdivision functionality. This will result in the selected marker showing up and sticking around until the user scrolls it out of focus.

Reply:
As long as on a delete the marker doesn't stick around.